doc/release-notes.md

```diff
@@ -81,6 +81,10 @@ RPC changes
    `fee`, `modifiedfee`, `ancestorfee` and `descendantfee`.
 - The new RPC `getzmqnotifications` returns information about active ZMQ
   notifications.
+- In `getmempoolancestors`, `getmempooldescendants`, `getmempoolentry` and
+  `getrawmempool` RPCs, to be consistent with the returned value and other RPCs
+  such as `getrawtransaction`, `vsize` has been added and `size` is now
+  deprecated.
```

Comment:
Should document that `size` will only be returned if bitcoind is started with -deprecatedrpc=size.